components/selectors/context.rs

```diff
@@ -49,6 +49,7 @@ pub enum VisitedHandlingMode {
 ///
 /// See: https://quirks.spec.whatwg.org/
 #[derive(PartialEq, Eq, Copy, Clone, Hash, Debug)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
```

Comment:
Given `selectors` doesn't have a `servo` feature, this is dead code right now. Could you revert the selectors changes?